scripts/train_alternating.py

```
@@ -0,0 +1,283 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import argparse
+import logging
+from typing import Dict, Any, Optional
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)